creator-node/src/config.js

```diff
@@ -570,6 +570,12 @@ const config = convict({
     env: 'maxNumberSecondsPrimaryRemainsUnhealthy',
     // 24 hours in seconds
     default: 86400
+  },
+  secondaryUserSyncDailyFailureCountThreshold: {
+    doc: 'Max number of sync failures for a secondary for a user per day before stopping further SyncRequest issuance',
+    format: 'nat',
+    env: 'secondaryUserSyncDailyFailureCountThreshold',
+    default: 10
```

Comment:
do we want to set this to a lower threshold for local dev/stage? for local dev, maybe we can set this to 1 as so we can test this more easily with a lower threshold.

Reply:
fair, done